var-core/src/main/java/io/varhttp/VarServletOutputStream.java

```diff
@@ -0,0 +1,28 @@
+package io.varhttp;
+
+import javax.servlet.ServletOutputStream;
+import javax.servlet.WriteListener;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+public class VarServletOutputStream extends ServletOutputStream {
+
+	public final ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
```

Comment:
make private and provide a getter?

Reply:
https://github.com/persequor-com/var-http/pull/69/commits/4d01b3b86e4518843ff9f2cd45984c14cfeb419c